File: Backend/ICE.Capa_Negocios/Modelos/Reporte.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ICE.Capa_Dominio.Modelos
{
    public class Reporte
    {
        public int Id { get; set; }
        //Datos que llena el supervisor
        public byte[]? MapaDeDescargas { get; set; }
        public string? Observaciones { get; set; }

        //Datos que llena el Tecnico de Linea
        public byte[]? Evidencia { get; set; }
        public string? ObservacionesTecnicoLinea { get; set; }
        //anidar las causas del frontend
        public string? Causas {get; set;}
        public DateTime? FechaHora { get; set; }
        
        // Relaciones con Informes
        public int InformeV1Id { get; set; }        

        public int InformeV2Id { get; set; }        

        public int InformeV3Id { get; set; }        

        public int InformeV4Id { get; set; }        

        public int Estado { get; set; }

        // Relación con Usuario Supervisor
        public int UsuarioSupervisorId { get; set; }        

        // Relación con Técnico de Línea
        public int TecnicoLineaId { get; set; }        
    }
}

File: frontend/reporteria-ice/src/app/Modelo/Reporte.ts
export interface Reporte {
    id: number,
    mapaDeDescargas?: string,
    observaciones?: string,
    evidencia?: string,
    observacionesTecnicoLinea?: string,
    causas?: string,
    fechaHora?: Date | null | string,
    informeV1Id: number,
    informeV2Id: number,
    informeV3Id: number,
    informeV4Id: number,
    usuarioSupervisorId: number,
    tecnicoLineaId: number,
    estado: 0
}

